### PR TITLE
Update renovate.json

### LINKED ISSUE #139: Adjust Renovate Config to be once a week, higher confidence level, keep on Node 18
🤔 I wonder if we can configure these to only open when the confidence level is higher 🤔

_Originally posted by @ann-kilzer in https://github.com/ourjapanlife/findadoc-server/issues/137#issuecomment-1419171651_

### PR DESCRIPTION
**Pull request by ann-kilzer**
Resolves #139 

# What changed
See inline comments

# Testing instructions
need to merge to main to test

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -3,5 +3,22 @@
     "extends": [
         "config:base"
     ],
-    "schedule": ["after 10am and before 5pm every weekday"]
+    "labels": ["package-update"],
```
Comment (ann-kilzer) [inline renovate.json:6]: Add a github label for easy filtering

File: renovate.json [hunk 2]
```diff
@@ -3,5 +3,22 @@
...
+    "schedule": ["after 10am every monday"],
```
Comment (ann-kilzer) [inline renovate.json:7]: Just once a week (Monday 7 pm JST) to reduce noise

File: renovate.json [hunk 3]
```diff
@@ -3,5 +3,22 @@
...
+    "stabilityDays": 14,
```
Comment (ann-kilzer) [inline renovate.json:8]: Wait 2 weeks for packages to be stable (can change after we deploy)

File: renovate.json [hunk 4]
```diff
@@ -3,5 +3,22 @@
...
+    "packageRules": [
+        {
+            "matchPackagePatterns": ["^eslint"],
```
Comment (ann-kilzer) [inline renovate.json:11]: Group eslint updates together

File: renovate.json [hunk 5]
```diff
@@ -3,5 +3,22 @@
...
+            "groupName": "eslint packages"
+        },
+        {
+            "matchPackagePatterns": ["node"]
```
Comment (ann-kilzer) [inline renovate.json:15]: Keep node at 18

File: renovate.json [hunk 6]
```diff
@@ -3,5 +3,22 @@
...
+            "allowedVersions": "/18.*/"
+        },
+        {
+            "matchDatasources": ["docker"],
```
Comment (ann-kilzer) [inline renovate.json:19]: Keep postgres (inside docker) at 14

**Review by ermish — APPROVED** [review]
looks awesome!

**Pull request merged**